registration/registration.go

```diff
@@ -168,6 +177,49 @@ func (r *Registration) CertifierInfo() *CertifierConfig {
 	return r.cfg.Certifier
 }
 
+func (r *Registration) LoadTrustedPublicKeys() error {
+	if r.cfg.Certifier == nil {
+		return ErrCertificationIsNotSupported
+	}
+
+	r.trustedKeysMtx.Lock()
+	defer r.trustedKeysMtx.Unlock()
+
+	r.trustedPublicKeys = [][]byte{}
```

Comment:
Should `trustedPublicKeys` be reset immediately? If the code below fails for some reason, the poet would be left w/o the previous keys. How about swapping the configured keys after reading the directory?